docs/user-guide.md

```diff
@@ -41,79 +39,79 @@ This guide provides detailed instructions on how to use Liberty Tools for the Ec
-* `export PATH=$PATH:../path/to/docker`
-* launch Eclipse via:   `PATH=$PATH:../path/to/docker eclipse.exe ...`
+* RUn the following command: `export PATH=$PATH:../path/to/docker`
+* launch Eclipse by running the following command  `PATH=$PATH:../path/to/docker eclipse.exe ...`
 
    
 ## Opening the Liberty dashboard view
 
-Click on the Open Liberty icon on the toolbar.
+Click the Open Liberty icon on the toolbar.
 
-![Dashboard context menu for Maven Project](images/openLibertyIconOnToolbar.png)
+![Dashboard context menu for Maven Project with Open Liberty icon](images/openLibertyIconOnToolbar.png)
 
-If the dashboard is opened and there are projects that already properly configured to run on Liberty and use Liberty dev mode, those projects are automatically added to the dashboard when it opens.
+If projects exist that are already properly configured to run on Liberty and use Liberty dev mode, those projects are automatically added to the dashboard when it opens.
 
-If you add new projects or make changes, and you need the dashboard to be refreshed, use the refresh icon in the Liberty dashboard toolbar.
+If you add new projects or make changes, and you need to refresh the dashboard, use the refresh icon in the Liberty dashboard toolbar.
 
 ![Dashboard Refresh highlighted](images/dashboardToolbarRefresh.png)
 
 
 ## Running your application on Liberty using dev mode
 
-There are 3 ways to start your Liberty application in dev mode by using the menu actions provided through the Liberty dashboard view or the Eclipse IDE explorer view.
+3 options are avaialble to start your Liberty application in dev mode by using the menu actions provided through the Liberty dashboard view or the Eclipse IDE explorer view.
 
 ### Using the Liberty dashboard view
 
-The Liberty dashboard provides a context menu to the Liberty projects in it. Users can choose the menu contents to perform a series of operations aiming to speed up application development.
+The Liberty dashboard provides a context menu to the Liberty projects in it. Users can choose the menu contents to perform a series of operations that can speed up application development.
 
 Maven Projects             | Gradle Projects
 :-------------------------:|:-------------------------:
 ![Dashboard context menu Gradle](images/maven-projectContextMenu.png) | ![Dashboard context menu Maven](images/gradle-projectContextMenu.png)
 
 ### Using the Project Explorer view
 
-The Liberty Tools plugin provides a set of launch shortcut actions to the Liberty projects shown in the explorer view. Users can choose these actions to perform a series of operations aiming to speed up application development.
+The Liberty Tools plugin provides a set of launch shortcut actions to the Liberty projects shown in the explorer view. Users can choose these actions to perform a series of operations that can speed up application development.
```

Comment:
I suggest adding a similar section for the explorer view, and then explaining that all options that are available from the dashboard can also be run from explorer view. Then this saves having to repetitively document both flows in every option. For example, see [this section of the Intellij doc](https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/docs/user-guide.md#use-the-go-to-action-intellij-idea-menu).

That doc focuses on the dashboard view and just mentions the alternative in that one section.

Reply:
OK, I tried a reorg see what you think.